modules/server_processes_manager/README.md

```diff
@@ -0,0 +1,65 @@
+# Server Processes Manager
+
+## Purpose
+
+The server processes manager module is used to get information about processes that were 
+launched asynchronously by other LORIS modules
+
+## Intended Users
+
+The only user that should have access to this module is the person in charge of
+the administration of the LORIS website
+
+## Scope
+
+This module displays the information relevant to the asynchronous processes
+(active and inactive) launched by other modules. The properties displayed for
```

Comment:
I'm not sure what "active and inactive" means in this context

Reply:
Replaced by "currently running or not"